<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddComment</class>
 <widget class="QDialog" name="AddComment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加评论</string>
  </property>
  <widget class="QGroupBox" name="gbStars">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>125</y>
     <width>400</width>
     <height>75</height>
    </rect>
   </property>
   <property name="title">
    <string>评价星级</string>
   </property>
   <widget class="QLabel" name="lbls3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>25</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbls5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>25</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbls4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>25</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbls2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>25</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbls1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>25</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbStars">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbContent">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>225</y>
     <width>400</width>
     <height>350</height>
    </rect>
   </property>
   <property name="title">
    <string>评价内容</string>
   </property>
   <widget class="QTextEdit" name="teContent">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>350</width>
      <height>300</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>back.png</normaloff>back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>next.png</normaloff>next.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCommit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>submit.png</normaloff>submit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbTitle">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>25</y>
     <width>225</width>
     <height>90</height>
    </rect>
   </property>
   <property name="title">
    <string>评论标题</string>
   </property>
   <widget class="QLineEdit" name="leTitle">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>40</y>
      <width>175</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>25</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>sbStars</sender>
   <signal>valueChanged(int)</signal>
   <receiver>AddComment</receiver>
   <slot>SetStars(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leTitle</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddComment</receiver>
   <slot>OnDelta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>474</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>teContent</sender>
   <signal>textChanged()</signal>
   <receiver>AddComment</receiver>
   <slot>OnDelta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbStars</sender>
   <signal>valueChanged(int)</signal>
   <receiver>AddComment</receiver>
   <slot>OnDelta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBack</sender>
   <signal>clicked()</signal>
   <receiver>AddComment</receiver>
   <slot>BackOne()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNext</sender>
   <signal>clicked()</signal>
   <receiver>AddComment</receiver>
   <slot>NextOne()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCommit</sender>
   <signal>clicked()</signal>
   <receiver>AddComment</receiver>
   <slot>OnCommit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SetStars(int)</slot>
  <slot>OnDelta()</slot>
  <slot>BackOne()</slot>
  <slot>NextOne()</slot>
  <slot>OnCommit()</slot>
 </slots>
</ui>
